conf/keycloak/Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN mvn clean package
 # ------------------------------------------
 # Stage 2: Build Keycloak Image
 # ------------------------------------------
-FROM quay.io/keycloak/keycloak:22.0
+FROM quay.io/keycloak/keycloak:25.0.6
```

Comment:
26.1.0 is current - any reason you chose this version instead?

Reply:
I just updated it to 26.1.0.

Initially, I used version 25, as the upgrade to version 26 involved some breaking changes. However, after adding some tweaks, it is now working good.